lua/lvim/core/nvimtree.lua

```diff
@@ -147,24 +147,11 @@ function M.setup()
     return
   end
 
-  local status_ok_1, utils = pcall(require, "nvim-tree.utils")
+  local status_ok_1 = pcall(require, "nvim-tree.utils")
   if not status_ok_1 then
     return
   end
```

Comment:
```suggestion
```

Reply:
cool ty.. not sure why i just didn't remove the whole function initially :-(